powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/CreateRecoveryCodeRequest.java

```diff
@@ -28,8 +32,17 @@
 @Data
 public class CreateRecoveryCodeRequest {
 
+    @Schema(description = "The identifier of the application")
+    @NotBlank(message = "Application ID must not be empty when creating recovery code")
     private String applicationId;
+
+    @Schema(description = "The identifier of the user")
+    @NotBlank(message = "User ID must not be empty when creating recovery code")
     private String userId;
+
+    @Schema(description = "The number of PUK codes")
```

Comment:
Suggestion to make it more clear
```suggestion
    @Schema(description = "The number of PUK codes to generate")
```